client/signup/config/step-components.js

```diff
@@ -12,11 +12,12 @@ const stepNameToModuleName = {
 	'creds-complete': 'creds-complete',
 	'creds-permission': 'creds-permission',
 	domains: 'domains',
-	emails: 'emails',
-	'domains-store': 'domains',
 	'domain-only': 'domains',
-	'domains-theme-preselected': 'domains',
 	'domains-launch': 'domains',
+	'domains-store': 'domains',
+	'domains-theme-preselected': 'domains',
+	'domains-with-email': 'domains',
+	emails: 'emails',
```

Comment:
I took the opportunity to sort these alphabetically

Reply:
I initially went with the exact same slug 😆 but it looked a bit strange in the address bar `start/onboarding-with-email/domains-with-email`, that's why I suggested `mailbox-domain`, but I suppose it isn't a huge deal :)

Reply:
Reevaluated this. It started to get a bit too weird when I was naming the `emails` step variation (`emails-with-email`..?)

So I went with your suggestion: the domains step is now `mailbox-domain` and the emails step will be `mailbox`